client/my-sites/sidebar-unified/index.jsx

```diff
@@ -34,6 +36,13 @@ export const MySitesSidebarUnified = ( { path } ) => {
 	const menuItems = useSiteMenuItems();
 	const isAllDomainsView = useDomainsViewStatus();
 	const isRequestingMenu = useSelector( getIsRequestingAdminMenu );
+	const [ collapsed, setCollapsed ] = useState( isWithinBreakpoint( '<960px' ) ? true : false );
```

Comment:
I wonder if you could set this based on a query param in the URL. Then you could sort of "share" the state of the sidebar from Calypso with WPAdmin.

Reply:
Great idea @getdave ! I think we can leave it for a next PR though?